### DISCUSSION TITLE
Can't bring up Desktop GUI

### DISCUSSION THREAD

**Discussion by mikeyates99** [Security-Onion-Solutions/securityonion#7209] [Unsupported Versions] [answered]
I created a Virtual Machine using the iso I downloaded from securityonionsolutions.com.  The version is securityonion-2.3.100-20220203

I followed the First time set up procedures outlined here: [https://docs.securityonion.net/en/2.3/first-time-users.html]

On the screen shots, it pops up to a Desktop GUI.  

When I try to run the command "sudo so-analyst-install"  I get an error message "failure "repodata/repoxml.xml from base: [Errno 256] No more mirrors to try
I will include a screenshot of the error.  It then repeats itself until it goes back to command line.  

I've tried the commands suggested on this page [https://docs.securityonion.net/en/2.3/installation.html#installation](url)
but none of the commands worked.  

Please help.  

Thank you for your time. 

<img width="665" alt="so-error1" src="https://user-images.githubusercontent.com/44930963/154143158-24708955-a7aa-455e-a5d1-b735c30f3e4f.png">

**Comment by njohnson-q** [comment]
Hi,

My first thought is an issue with DNS. If you did run through the so-setup-network it should have setup correctly.

A few questions/suggestions to get some more information:
1. Is this an analyst install only or is analyst on top of of another node?
2. Is networking setup on the host?
3. Try doing an nslookup on a website or ping (something like "ping www.google.com" or "nslookup www.google.com")


Thank you

**Reply by mikeyates99** [reply]
During my reinstall, I received an error during the setup that I had also received the first time, only I had forgotten about it.  Here's a screenshot.  

I'm far from reproach, but I know how to configure an internet connection.  There is no IP conflict.  I've triple checked every octet.  It's a simple CIDR /24.  I don't know why this machine is pretending to not get online.  All of my other VMs are online just fine.  I know they are separate machines, but still....
<img width="328" alt="so-error2" src="https://user-images.githubusercontent.com/44930963/154154025-079ef6cc-b223-48d7-b42a-811a16090ffc.png">
.

**Reply by dougburks** [reply]
From the VM, are you able to ping the default gateway for your Internet connection?

**Comment by mikeyates99** [comment]
Thank you, so much, for your responses.  I went back and read some more documentations, and learned that I used a different Linux system in my Virtual Machine setup,  So, I'm reinstalling the machine to perfectly match everything described.  
As far as the DNS possibility, I'm using a simple wifi for internet.  The DNS is 8.8.8.8 and 8.8.4.4.  I'm able to ping everything from the local machine.  I could try a different DNS, though.
I'm installing security onion on my blank VM (using NAT and bridged for the network adapters) on my Windows 10 Pro Laptop.  Running VM Workstation Pro.  

Mike
    On Tuesday, February 15, 2022, 02:46:50 PM CST, Nick Johnson ***@***.***> wrote:  
 
 


Hi,

My first thought is an issue with DNS. If you did run through the so-setup-network it should have setup correctly.

A few questions/suggestions to get some more information:
   
   - Is this an analyst install only or is analyst on top of of another node?
   - Is networking setup on the host?
   - Try doing an nslookup on a website or ping (something like "ping www.google.com" or "nslookup www.google.com")

Thank you

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by mikeyates99** [comment, marked as answer]
The solution I found was to install VMWare on a different computer and instill the iso from there.  Everything worked fine.  I have no idea why, because that laptop is running other VMs without any network issues.  However, I am now up and running.  I appreciate the time and effort from everyone.  
    On Wednesday, February 16, 2022, 07:55:46 AM CST, Doug Burks ***@***.***> wrote:  
 
 


From the VM, are you able to ping the default gateway for your Internet connection?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
